apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx

```diff
@@ -40,14 +42,14 @@ Ensure that you have installed the VTEX IO CLI on your machine. This CLI is requ
 </Callout>
 
 ```bash copy
```

Comment:
For some reason, a couple code lines has this extra indentation at the beginning for the line. Don't know if it's related to the indentation on the `mdx` file. 🫠
<img width="958" alt="image" src="https://github.com/vtex/faststore/assets/11613011/add7f990-71a5-4edd-badc-37b3650a6558">